### DISCUSSION TITLE
how can i get pro version by universty code

### DISCUSSION THREAD

**Discussion by MoARABY** [community/community#112967] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

>>>>

**Comment by MoARABY** [comment, marked as answer]
i was have the link but is not working

**Comment by MoARABY** [comment]
should be inside the collage building

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @MoARABY! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
